gui

```diff
@@ -157,6 +157,12 @@ runonce <<"EOF"
   
   refresh_all_pkgapp_status
 EOF
+
+# change settings icon
+runonce <<"EOF"
+  [ -a "$HOME/.local/share/applications/pi-apps-settings.desktop" ] && sed -i "s/logo.png/settings.png/g" "$HOME/.local/share/applications/pi-apps-settings.desktop" || true
```

Comment:
Why are you using the `-a` flag instead of the `-f` flag here?

Reply:
From `help test`:

```
      -a FILE        True if file exists.
      -f FILE        True if file exists and is a regular file.
```

While `"$HOME/.local/share/applications/pi-apps-settings.desktop"` will mostly be a regular file, I don't think there is any differences between `-a` and `-f`.